sys/geom/union/g_union.c

```diff
@@ -358,6 +358,7 @@ g_union_ctl_create(struct gctl_req *req, struct g_class *mp, bool verbose)
 fail1:
 	g_destroy_consumer(lowercp);
 	g_destroy_provider(newpp);
+	g_free(sc);
```

Comment:
Presumably we should also `rw_destroy()` the lock that was initialized within the softc?